modules/service/src/main/resources/db/migration/V1041__flamingos2_longslit_telluric_type.sql

```diff
@@ -0,0 +1,11 @@
+
+ALTER TABLE t_flamingos_2_long_slit
+ADD COLUMN c_telluric_type jsonb NOT NULL DEFAULT '{"tag":"HOT","starTypes":null}'::jsonb;
```

Comment:
Json fields always seem a bit frightening to me because they seem like they have maintenance implications. We do use them occasionally though.  I don't know what the best approach is though.  If we added a telluric tag enum and a tag field of that type, at least that bit would be easier to track?  I'm not opposed to this though, just unsure.

Reply:
Are there any rules about which long slit observations can and cannot have a telluric type?